File: cfgs/pythoncfgs/COMPUTE_N_UNDER_MODULO_P.py
Nodes
1;conditional;foo_1 >= foo_2;8;
2;statement;foo_1 = 1;10;
3;conditional;foo < foo;11;
4;statement;return foo_1;13;
5;statement;foo_1 = foo_2 * foo_3 % foo_4;12;
6;statement;return 0;9;
7;exit;;
Edges
0;2;6;
1;3;
2;4;5;
3;7;
4;3;
5;7;
6;

File: cfgs/java_cfgs_foo/COMPUTE_N_UNDER_MODULO_P.java
Nodes
1;special;;;
2;conditional;foo_1 >= foo_2;14;
3;statement;return 0;14;
4;exit;;;
5;statement;int result = 1;15;
6;statement;int i = 1;16;
7;conditional;foo_1 <= foo_2;17;
8;statement;result = (foo_1 * foo_2) % foo_3;18;
9;statement;i++;18;
10;statement;return foo_1;19;
Edges
0;2;
1;3;5;
2;4;
3;
4;6;
5;7;
6;8;10;
7;9;
8;7;
9;4;
